docs/commands.md

```diff
@@ -471,7 +471,8 @@ _Arguments_
 * _ID_: [Session](glossary.html#session) unique ID (if not given, RedisGears will generate one)
 * _DESCRIPTION_: Optional [Session](glossary.html#session) description
 * _UPGRADE_: If the session with this name already exists, replace it.
-* _REPLACE_WITH_: Set the new [Session]() as a replacement of the session give by this argument.
+* _REPLACE_WITH_: Set the new [Session](glossary.html#session) as a replacement of the session give by this argument.
+* _FORCE_REQUIREMENTS_REINSTALLATION_: Force reinstallation of all the requiments. When this option is used RedisGears will reset the python interpreter modules cache after the re-installation is finished. This way the modules will be reloaded by the python interpreter. One important limitation of using this option is that **It is not possible to have multiple version of the same requirement at the same time**, if one registration used an old version of a requirement and another registration upgrade it, then after restart (or even before restart on some rare cases) they will both have the new version and if the new version is **not** backward compatible **the code might break with errors**. This option is available sense v1.2.4.
```

Comment:
Any chance this can be renamed to FORCE_REINSTALL_REQUIREMENTS? It's more natural.